server/services/unusedDeductionsService.ts

```diff
@@ -48,36 +58,36 @@ export default class UnusedDeductionsService {
         if (calculatedUnusedDeducions || calculatedUnusedDeducions === 0) {
           const dbDeductions = this.getTotalUnusedRemand(adjustments)
           if (calculatedUnusedDeducions === dbDeductions) {
-            return true
+            return 'NONE'
           }
           await delay(this.waitBetweenTries)
           adjustments = await this.adjustmentsService.findByPersonOutsideSentenceEnvelope(nomsId, token)
           // Try again
         } else {
           // Unable to calculate unused deductions.
-          return false
+          return 'NONE'
```

Comment:
This is where the calculated unused deductions don't match whats in the database. There isn't really a message for this.